kvmagent/kvmagent/plugins/prometheus.py

```
@@ -1002,7 +1002,7 @@ def collect_ipmi_state():
         return collect_equipment_state_last_result
 
     # get ipmi status
-    metrics['ipmi_status'].add_metric([], bash_r("ipmitool mc info"))
+    metrics['ipmi_status'].add_metric([], bash_r("ipmitool mc info", timeout=30))
```

Ruff (0.15.7)

[error] `bash_r` may be undefined, or defined from star imports

(F405)